integrations/gcp/gcp_core/errors.py

```diff
@@ -0,0 +1,13 @@
+from port_ocean.exceptions.core import OceanAbortException
+
+
+class GotFeedCreatedSuccessfullyMessageError(OceanAbortException):
+    pass
+
+
+class AssetHasNoProjectAncestorError(OceanAbortException):
+    pass
+
+
+class ResourceNotFoundError(Exception):
```

Comment:
I don't want ocean to abort on missing resource in real time, since there is a short delay between the event of creation arriving and it's data being present in the inventory.